jme3-core/src/main/java/com/jme3/anim/tween/action/Action.java

```diff
@@ -105,4 +192,29 @@ public void cloneFields(Cloner cloner, Object original) {
         actions = cloner.clone(actions);
         mask = cloner.clone(mask);
     }
+    
+    /**
+     * Tests the value of the forward flag.
+     *
```

Comment:
I feel that the API user does not need to know/care that there is a flag involved. The flag variable is an implementation detail that is hidden by the API. Why expose it in the documentation?

Suggestion: `Test whether the Action is running in "forward" mode.`

Reply:
There is this issue #2021, we can discuss it there.

> Suggestion: `Test whether the Action is running in "forward" mode.`
Yeah, sure, I think it's better.